app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -70,11 +71,25 @@ def authorization_params
       params.permit(OpenidConnectAuthorizeForm::ATTRS)
     end
 
+    def handle_invalid_logged_out_form
+      return if current_user
+
+      result = @authorize_form.check_submit
+      return if result.success?
+
+      track_authorize_analytics(result)
```

Comment:
To me, it's the same event, they attempted and had a bad form so we can't complete the request.